src/transformers/models/clip/configuration_clip.py

```diff
@@ -199,6 +199,7 @@ def __init__(
         intermediate_size=3072,
         num_hidden_layers=12,
         num_attention_heads=12,
+        num_channels=3,
```

Comment:
What's the reason this attribute is added?

Reply:
The CLIP backbone was previously hardcoded to process only RGB images, this is an additional attribute to process greyscale input

Reply:
Ok, although I'm not seeing `config.num_channels` used in `modeling_clip.py`

Reply:
Probably to be used when defining `self.patch_embedding`?